### PR TITLE
Update "testing" build script so it works regardless of execution location

### PR DESCRIPTION
**Pull request by cailyoung**
I updated `build-commandpost-testing.sh` so that it:
* Mirrors the method the release script uses to figure out which directories to look in
* Uses `librelease.sh` to drive the build process so it can be altered later if necessary

### CODE REVIEW

File: scripts/build_commandpost_testing.sh [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
+
+# SCRIPT_NAME="$(basename "$0")"
+SCRIPT_HOME="$(dirname "$(greadlink -f "$0")")" # fully qualified directory of this script
+HAMMERSPOON_HOME="$(greadlink -f "${SCRIPT_HOME}/../../CommandPost-App")" # fully qualified directory of the CommandPost-App repository
 
-make clean
-make release
-make docs
+# Import our function library
+# shellcheck source=scripts/inc/librelease.sh disable=SC1091
```
[the remark this answers is not in the document]
Comment (cailyoung) [reply scripts/build_commandpost_testing.sh:19]: Stickler needs to update their shellcheck version to one that includes the source argument for directives. All this does is tell shellcheck where the script actually is.

[the remark this answers is not in the document]
Comment (cailyoung) [reply scripts/build_commandpost_testing.sh:19]: Same comment as above.

[the remark this answers is not in the document]
Comment (cailyoung) [reply scripts/build_commandpost_testing.sh:19]: Opened https://github.com/markstory/lint-review/issues/172

**Comment by latenitefilms** [comment]
It doesn't look like there's anything in your script to change over to the "test" configuration?

**Comment by cailyoung** [comment]
The original one was executing `make release` which builds the Xcode project in a release config. You could alter my new function in `librelease.sh` to do a debut scheme build but I don’t know if that’s necessary for CommandPost dev work?

**Comment by cailyoung** [comment]
*debug

**Comment by cailyoung** [comment]
@latenitefilms are you saying you'd prefer the script to actually build CommandPost-App from the Debug scheme? Thats an easy fix.

**Comment by cailyoung** [comment]
@latenitefilms this now passes stickler. Let me know if you want the xcodebuild to run against the debug scheme instead.

**Comment by latenitefilms** [comment]
@cailyoung - To be honest, I'm not really sure how the debug/test scheme works. Need to do some playing around with Hammerspoon.

**Pull request merged**